plugins/innogrit/innogrit-nvme.c

```diff
@@ -0,0 +1,529 @@
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include "common.h"
+#include "nvme.h"
+#include "libnvme.h"
+#include "nvme-print.h"
+#include "typedef.h"
+
+#define CREATE_CMD
+#include "innogrit-nvme.h"
+
+
+/**********************************************************************
+*
+* Entry Point: innogrit_list
+*
+***********************************************************************/
+static int innogrit_list(int argc, char **argv, struct command *command, struct plugin *plugin)
```

Comment:
Why do you need your own vendor 'list' command? What's wrong with the generic one?

Reply:
becacuse vendor command only support innogrit disk, we want only list innogrit disk